src/gps/ubx.h

```diff
@@ -10,19 +10,19 @@ const uint8_t GPS::_message_CFG_RXM_PSM[] PROGMEM = {
     0x01  // Power save mode
 };
 
+// only for Neo-6
 const uint8_t GPS::_message_CFG_RXM_ECO[] PROGMEM = {
     0x08, // Reserved
     0x04  // eco mode
 };
 
 const uint8_t GPS::_message_CFG_PM2[] PROGMEM = {
-    0x01, // version
-    0x00, // Reserved 1, set to 0x06 by u-Center
-    0x00, // Reserved 2
-    0x00, // Reserved 1
-    //    0x0E, 0x81, 0x43, 0x01, // flags-> cyclic mode, wait for normal fix ok, do not wake to update RTC or EPH, doNotEnterOff,
-    //    LimitPeakCurrent
-    0x00, 0x01, 0x03, 0x00, // flags
+    0x01,                   // version
+    0x00,                   // Reserved 1, set to 0x06 by u-Center
+    0x00,                   // Reserved 2
+    0x00,                   // Reserved 1
+    0x00, 0x11, 0x03, 0x00, // flags-> cyclic mode, wait for normal fix ok, do not wake to update RTC or EPH, doNotEnterOff,
```

Comment:
The flags here are provided by U-Center. Are you basing these changes on only flipping the bits that u-center flips when you toggle the flags in the Message View hex previewer?

Reply:
My testing conditions are with an external antenna (u-blox ANN-MB-00-00) on the roof with a clear view of the sky. Connected to a Neo-6 module connected to my Chatter 2 esp32 based board. I have a Nordic Power Profiler Kit 2 connected between the Chatters 3.3v supply and the Neo-6. For testing with u-center I have the same module connected to a serial-> USB adapter connected to a loptop running u-center 23.08.